### DISCUSSION TITLE
Docker Install vs Desktop App (and Connecting the Two)

### DISCUSSION THREAD

**Discussion by drinksomewhiskey** [toeverything/AFFiNE#7420] [General]
Hi,

I was able to install via docker and set up the local database. 

I also installed the desktop app for Mac OS. 

When using the magic link option during the sing in process, the docker and desktop apps sign me in, but the accounts appear to be two separate accounts even though the same email was used to sign in. Is this intended behavior? In terms of use credentials, both share the same email but have different passwords. There seems to be two accounts.

For context, I enabled cloud on the local database running via docker. I also tried using email / password instead of magic link. The "docker account" wont work on the desktop app and the "desktop account" wont work in docker. Both cases state that the password is incorrect, but they work in their respective apps.

I am mainly wondering if theres any way to sign into the docker database (self hosted on my server) via the desktop app. If not, how can I have two accounts under the same email address? Could it be that the account for the docker instance is just a local account even though I selected enable cloud sync via the docker instance?

Lastly, if this is intended behavior, is there any word around linking the desktop app (and future mobile app) to self hosted servers?

Thank you.

**Comment by dchakro** [comment]
Were you able to figure this out? I started out by self hosting affine today and it worked fine on the browser, but I can't figure out how to connect the desktop (mac) app to the local instance I'm running.

**Reply by drinksomewhiskey** [reply]
I did not. I have a feeling that these are independent and that the desktop apps don't have a way to connect to local databases. However, no one has responded to confirm my suspicions.

**Comment by diod2022** [comment]
HI,

Any updates on this matter?

**Comment by robinransom-fk** [comment]
Please Follow -> #2646
